app/Helpers/database/leaderboard.php

```diff
@@ -157,10 +157,15 @@ function GetLeaderboardData(
         }
     }
 
+    if ($numToFetch == 0) {
```

Comment:
```suggestion
    if ($numToFetch === 0) {
```
Similar to JS, better to use triple equals with PHP.